src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingDetails.java

```diff
@@ -24,10 +24,6 @@ public class HearingDetails {
 
     private HearingWindow hearingWindow;
 
-    private Number duration;
-
-    private List<String> nonStandardHearingDurationReasons;
-
```

Comment:
Why have you removed these? These are set by hmc?

Reply:
I have restored them